This reverts an upstream change where the IDs for ImGui's docking
nodes were changed, which bricks existing OpenSim Creator imgui.ini
files that users have already cultivated.

---

diff --git a/third_party/imgui/imgui.cpp b/third_party/imgui/imgui.cpp
index 573efe244..db7856ccc 100644
--- a/third_party/imgui/imgui.cpp
+++ b/third_party/imgui/imgui.cpp
@@ -19668,7 +19668,7 @@ ImGuiID ImGui::DockSpaceOverViewport(ImGuiID dockspace_id, const ImGuiViewport*
         host_window_flags |= ImGuiWindowFlags_NoMouseInputs;
 
     char label[32];
-    ImFormatString(label, IM_ARRAYSIZE(label), "WindowOverViewport_%08X", viewport->ID);
+    ImFormatString(label, IM_ARRAYSIZE(label), "DockSpaceViewport_%08X", viewport->ID);  // opensim creator patch: backwards compat with old imgui.ini files
 
     PushStyleVar(ImGuiStyleVar_WindowRounding, 0.0f);
     PushStyleVar(ImGuiStyleVar_WindowBorderSize, 0.0f);
